pkgs/tools/misc/wacomtablet-kcm/default.nix

```
@@ -0,0 +1,32 @@
+{
+  stdenv, mkDerivation, fetchurl, extra-cmake-modules, qtx11extras,
+  plasma-workspace, libwacom, xf86_input_wacom
+}:
+
+mkDerivation {
+  name = "wacomtablet-kcm";
```

Use pname instead of name here

```suggestion
  pname = "wacomtablet-kcm";
```